docs/src/extend/custom-rules.md

```diff
@@ -340,21 +366,22 @@ Best practices for fixes:
 1. Make fixes as small as possible. Fixes that are unnecessarily large could conflict with other fixes, and prevent them from being applied.
 1. Only make one fix per message. This is enforced because you must return the result of the fixer operation from `fix()`.
 1. Since all rules are run again after the initial round of fixes is applied, it's not necessary for a rule to check whether the code style of a fix will cause errors to be reported by another rule.
-    * For example, suppose a fixer would like to surround an object key with quotes, but it's not sure whether the user would prefer single or double quotes.
 
-        ```js
-        ({ foo : 1 })
+* For example, suppose a fixer would like to surround an object key with quotes, but it's not sure whether the user would prefer single or double quotes.
+
+  ```js
+  ({ foo : 1 })
```

Comment:
I think this was fine because the example relates to the last item only.

Reply:
> it's still indented to be scoped to the last item only.

It doesn't look like that to me, here's before:

![image](https://user-images.githubusercontent.com/44349756/229946489-959b0fed-9e3d-49e8-8835-cfa95b904ad2.png)

and after:

![image](https://user-images.githubusercontent.com/44349756/229946651-6d76f455-1a48-4644-9848-a3239b2a6220.png)

Reply:
ah ok, i probably only checked in the vscode markdown preview where it works as expected. funky behavior on the part of the 11ty parser, but anyways...fixed now.